package.json

```diff
@@ -107,6 +107,7 @@
     "angular-route": "^1.7.9",
     "bootstrap": "^3.4.1",
     "cropperjs": "^1.5.6",
+    "d3-hexbin": "^0.2.2",
```

Comment:
Could you also move d3 to this section (from `"devDependencies"` to `"dependencies"`). Thanks!

Reply:
Done.